k8s/namespaces/wa/wa-task-batch-reconfiguration/aat-01.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-reonfiguration
```

Comment:
typo

Reply:
Fixed in https://github.com/hmcts/cnp-flux-config/pull/16179